internal/cmd/subcmds/pluginFlag.go

```diff
@@ -40,7 +40,7 @@ func (f *PluginFlag) BuildPlugins(verbose bool) ([]shared.RuleSet, error) {
 		client := plugin.NewClient(&plugin.ClientConfig{
 			HandshakeConfig: shared.Handshake,
 			Plugins:         shared.PluginMap,
-			Cmd:             exec.Command("sh", "-c", value),
+			Cmd:             exec.Command(value),
```

Comment:
[memo] It works by directly passing an executable.

Reply:
I remember now that `sh -c` is necessary when giving flags to a plugin like below:

```
# it works now but doesn't work after this fix.
$ ./protolint -plugin "./plugin_example -go_style=false" _example/proto/
```

Reply:
Could you expand the flags such that they are passed as arguments to the command, instead of simply passing `value`?

Naive approach would be to split on spaces, but that would break in the case of the flags containing quoted arguments containing spaces. Writing a quick parser to support this shouldn't be too difficult.

Reply:
Thank you for your nice suggestion.
I'm interested in that go lib.

As I'm not a Windows user, I've been waiting for others' feedbacks, like https://github.com/yoheimuta/protolint/issues/144#issuecomment-761749244. These would be used for considering this fix's priority because I have no enough time.